src/App.jsx

```diff
@@ -14,57 +15,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
```

Comment:
It is possible to bring out the `clearSelection` function separately, which will improve readability.
```
const clearSelection = () => setSelectedGood('');
```